src/cli/cli.ts

```diff
@@ -243,7 +243,7 @@ if (!process.env.PW_CLI_TARGET_LANG) {
 if (process.argv[2] === 'run-driver')
   runDriver();
 else if (process.argv[2] === 'run-server')
-  runServer(process.argv[3] ? +process.argv[3] : undefined);
+  runServer(process.argv[3] ? +process.argv[3] : undefined, process.argv?.[4]).catch(logErrorAndExit);
```

Comment:
nit: why `argv?` - could it be undefined?

Reply:
Good point, fixed.